sail cannot be removed

As https://github.com/rems-project/sail/blob/sail2/INSTALL.md said, use `opam pin remove sail` will get something like this and sail still there.

```
[ariel@archlinux sail]$ opam reinstall sail                                                                                                                                                     
                                                                                                                                                                                                
<><> Synchronising pinned packages ><><><><><><><><><><><><><><><><><><><><><><>                                                                                                                
[sail.0.19.1] synchronised (no changes)                                                                                                                                                         
                                                                                                                                                                                                
The following actions will be performed:                                                                                                                                                        
=== recompile 1 package                                                                                                                                                                         
  ↻ sail 0.19.1 (pinned)                                                                                                                                                                        
                                                                                                                                                                                                
<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>                                                                                                                
⊘ removed   sail.0.19.1                                                                                                                                                                         
∗ installed sail.0.19.1                                                                                                                                                                         
Done.
[ariel@archlinux sail]$ opam pin remove sail
Ok, sail is no longer pinned to git+file:///home/ariel/sail/sail#pr/cJSON (version 0.19.1)
The following actions will be performed:
=== recompile 1 package
  ↻ sail 0.19.1

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
⬇ retrieved sail.0.19.1  (cached)
[ERROR] The compilation of sail.0.19.1 failed at "dune build -p sail -j 15 --promote-install-files=false @install".

#=== ERROR while compiling sail.0.19.1 ========================================#
# context     2.4.1 | linux/x86_64 | ocaml-base-compiler.5.2.1 | https://opam.ocaml.org#c44248fe760182a13f686e82e49611e36beb937c
# path        ~/.opam/5.2.1/.opam-switch/build/sail.0.19.1
# command     ~/.opam/opam-init/hooks/sandbox.sh build dune build -p sail -j 15 --promote-install-files=false @install
# exit-code   1
# env-file    ~/.opam/log/sail-27594-3bc677.env
# output-file ~/.opam/log/sail-27594-3bc677.out
### output ###
# (cd _build/default && /home/ariel/.opam/5.2.1/bin/ocamlopt.opt -w -40 -g -I src/bin/.sail.eobjs/byte -I src/bin/.sail.eobjs/native -I /home/ariel/.opam/5.2.1/lib/dune-private-libs/dune-section -I /home/ariel/.opam/5.2.1/lib/dune-site -I /home/ariel/.opam/5.2.1/lib/dune-site/private -I /home/ariel/.opam/5.2.1/lib/lem -I /home/ariel/.opam/5.2.1/lib/lem_zarith -I /home/ariel/.opam/5.2.1/lib/libs[...]
# File "src/bin/sail.ml", line 257, characters 30-53:
# 257 |       ("-list_files", Arg.Set Frontend.opt_list_files, " list files used in all project files");
#                                     ^^^^^^^^^^^^^^^^^^^^^^^
# Error: This expression has type "string option ref"
#        but an expression was expected of type "bool ref"
#        Type "string option" is not compatible with type "bool"



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
┌─ The following actions failed
│ λ build sail 0.19.1
└─ 
╶─ No changes have been performed
[ariel@archlinux sail]$ which sail
/home/ariel/.opam/5.2.1/bin/sail
[ariel@archlinux sail]$ ls /home/ariel/.opam/5.2.1/bin/sail
/home/ariel/.opam/5.2.1/bin/sail
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -54,19 +54,20 @@ repository. Assuming you have previously followed the above
 instructions (required to install dependencies):
 ```
 git clone https://github.com/rems-project/sail.git
-opam pin add sail
+cd sail
+opam pin add .
 ```
 will install from a local checkout of the Sail sources.
 
-You can update with new changes as they are committed by pulling and reinstalling:
+You can update with new changes as they are committed by pulling and reinstalling in the `sail` directory:
 ```
 git pull
-opam reinstall sail
+opam reinstall .
 ```
 
-To remove the pin and revert to the latest released opam package type:
+To remove the pin and revert to the latest released opam package type in the `sail` directory:
 ```
-opam pin remove sail
+opam pin remove .
 ```
 
 ### Building from source (without opam)

